action/main.py

```diff
@@ -25,6 +25,6 @@
 # If CHANGED_FILES is not empty, split it into a list; otherwise, use SRC
 files_to_check = shlex.split(CHANGED_FILES or SRC)
 
-proc = run(["pipx", "run", req, *shlex.split(ARGS), *files_to_check])
+proc = run(["pipx", "run", req, "check", *shlex.split(ARGS), *files_to_check])
```

Comment:
This would break the config using the format check [as described in the README](https://github.com/ChartBoost/ruff-action?tab=readme-ov-file#use-ruff-format).
As a matter of fact, things work just fine with the default (no args given) because the default for args is "check". So presumably, your problem is that you're setting args without it containing "check".